docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: how-it-works
+title: How Temporal Works
+sidebar_label: How It Works
+description: This guide provides a short overview of How Temporal Works.
+toc_max_heading_level: 4
+keywords:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+
+tags:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+---
+
+Temporal provides a comprehensive platform for building distributed applications with a focus on durability and reliability. 
+
+## Resilient applications by default
+
+Temporal ensures your applications are reliable by default, handling everything from minor network outages to significant issues like unexpected reboots, or hardware failures.
+By offloading the responsibility of failure management from the application to the platform, this removes the need for extensive coding, testing, and maintenance tasks. 
+
+This shift simplifies the development process and enables your application to seamlessly recover from disruptions. 
+It does so by automatically reconstructing its previous state. 
+Your application logic can be recovered, replayed, or paused from any previous point in the application process.
+
+
+## How It Works
```

Comment:
Consider the title of the page is already "How it works" - do we need this header here as well?